tests/test-runner/bin/zts-report.py.in

```diff
@@ -191,6 +191,20 @@ elif sys.platform.startswith('linux'):
         'casenorm/sensitive_formd_lookup': ['FAIL', '7633'],
         'casenorm/sensitive_formd_delete': ['FAIL', '7633'],
         'removal/removal_with_zdb': ['SKIP', known_reason],
+        'alloc_class/alloc_class_009_pos': ['FAIL', '12848'],
+        'alloc_class/alloc_class_010_pos': ['FAIL', '12848'],
+        'alloc_class/alloc_class_011_neg': ['FAIL', '12848'],
+        'cli_root/zfs_copies/zfs_copies_006_pos': ['FAIL', '12848'],
+        'cli_root/zpool_import/import_rewind_device_replaced': ['FAIL', '12848'],
+        'history/history_006_neg': ['FAIL', '12848'],
+        'history/history_008_pos': ['FAIL', '12848'],
+        'mmp/mmp_inactive_import': ['FAIL', '12848'],
+        'no_space/enospc_002_pos': ['FAIL', '12848'],
+        'refreserv/refreserv_raidz': ['FAIL', '12848'],
+        'rsend/rsend_007_pos': ['FAIL', '12848'],
+        'rsend/rsend_010_pos': ['FAIL', '12848'],
+        'rsend/rsend_011_pos': ['FAIL', '12848'],
+        'snapshot/rollback_003_pos': ['FAIL', '12848'],
```

Comment:
After recent changes it looks like we could whittle this down to.

```
         'cli_root/zpool_import/import_rewind_device_replaced': ['FAIL', '12848'],
         'mmp/mmp_inactive_import': ['FAIL', '12848'],
         'no_space/enospc_002_pos': ['FAIL', '12848'],
         'refreserv/refreserv_raidz': ['FAIL', '12848'],
         'snapshot/rollback_003_pos': ['FAIL', '12848'],
```

Reply:
Sure, let's see how this goes.

Reply:
I did a little investigation and opened PRs for the `refreserv_raidz` and `refreserv_raidz` failures.  We should be able to shed those two as well, then I think this is good to go.

Reply:
Shed! (Presuming the second one was rollback_003.)

Reply:
Right.  I _think_ we just need the following.  The first two aren't easy fixes, and the `enospc_002_pos` test probably just needs someone to go over it carefully and see where it's possible to go off the rails.

```
         'cli_root/zpool_import/import_rewind_device_replaced': ['FAIL', '12848'],
         'mmp/mmp_inactive_import': ['FAIL', '12848'],
         'no_space/enospc_002_pos': ['FAIL', '12848'],
```

Reply:
I think that I've just pushed what you requested.